be/src/vec/exec/format/json/new_json_reader.cpp

```diff
@@ -1524,7 +1524,6 @@ Status NewJsonReader::_simdjson_parse_json(bool* is_empty_row, bool* eof) {
 }
 Status NewJsonReader::_simdjson_parse_json_doc(size_t* size, bool* eof) {
     // read a whole message
-    SCOPED_TIMER(_file_read_timer);
```

Comment:
If we remove this profile, some info will lost.Maybe we could add it somewhere else